### PR TITLE
WIP: Switched Splash Screen to Tenacity Logo

### PR DESCRIPTION
**Pull request by caughtquick**
Signed-off-by: caughtquick <me@caughtquick.tech>

Resolves: *#223*

*(short description of the changes and the motivation to make the changes)*

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [ ] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

### CODE REVIEW

File: src/SplashDialog.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ SplashDialog::SplashDialog(wxWindow * parent)
 void SplashDialog::OnChar(wxMouseEvent &event)
 {
    if ( event.ShiftDown() && event.ControlDown() )
-      wxLaunchDefaultBrowser("https://www.audacityteam.org");
+      wxLaunchDefaultBrowser("https://www.m.org");
```
Comment (fitojb) [inline src/SplashDialog.cpp]: Please fix this URL.

File: src/SplashDialog.cpp [hunk 2]
```diff
@@ -46,9 +46,9 @@ most commonly asked questions about Audacity.
 
 // DA: Logo for Splash Dialog (welcome dialog)
 #ifdef EXPERIMENTAL_DA
-#include "../images/DarkAudacityLogoWithName.xpm"
+#include "../images/DarkTenacityLogoWithName.xpm"
```
Comment (fitojb) [inline src/SplashDialog.cpp]: "DarkTenacity" is not a thing; don't just blindly replace "Audacity" here

Comment (caughtquick) [reply src/SplashDialog.cpp]: It does exist, I added it.

Comment (fitojb) [reply src/SplashDialog.cpp]: It doesn't exist *as a project*; that's what I meant. I wasn't talking about the file. See www.darkaudacity.com

Comment (caughtquick) [reply src/SplashDialog.cpp]: Yes, that's for the DarkTenacity [logo](https://github.com/tenacityteam/assets).

**Pull request closed without merging**